### PR TITLE
Improve GetBalance unit test.

### PR DESCRIPTION
**Pull request by Spl3en**

### CODE REVIEW

File: IconSDK.Tests/TestRPC.cs [hunk 1]
```diff
@@ -13,8 +14,22 @@ public class TestRPC
...
         {
             var GetBalance = new GetBalance(API_URL);
+
+            // case 0: Successfull calls
+            // Get balance from EOA or score successfully
             var balance = await GetBalance.Invoke("hx0000000000000000000000000000000000000000");
-            Console.WriteLine($"Treasury : {balance.ToString()}");
+            Assert.IsInstanceOf<System.Numerics.BigInteger>(balance);
+            // No prefix
+            var balance2 = await GetBalance.Invoke("0000000000000000000000000000000000000000");
```
Comment (Spl3en) [inline IconSDK.Tests/TestRPC.cs:23]: I'm actually surprised that `GetBalance("0000000000000000000000000000000000000000")` returns the balance correctly. I though icon-service or loopchain nodes would have rejected such address without any "hx" or "cx" prefix, but they do not (also confirmed with the icon-sdk-python)

File: IconSDK.Tests/TestRPC.cs [hunk 2]
```diff
@@ -13,8 +14,22 @@ public class TestRPC
...
+            Assert.True(balance == balance2);
+
+            // case 1: when a param is wrong.
+            // Wrong hex format
+            Assert.ThrowsAsync<FormatException>(() => GetBalance.Invoke("hxWRONG"));
+            // Wrong prefix
+            Assert.ThrowsAsync<FormatException>(() => GetBalance.Invoke("0x0000000000000000000000000000000000000000"));
+            // Wrong number of bytes
+            Assert.ThrowsAsync<Exception>(() => GetBalance.Invoke("hx0000000000000000000"));
```
Comment (Spl3en) [inline IconSDK.Tests/TestRPC.cs:32]: Shouldn't it also throw a `FormatException` ?

File: IconSDK.Tests/TestRPC.cs (hunk 1 above)
Comment (zsaladin) [reply IconSDK.Tests/TestRPC.cs:23]: As you know an address consists of bytes. So I decided that I make it casted to 'ExternalAddress' object implicitly if its format is fit with hex string. The SDK treats addresses as 'ExternalAddress' or 'ContractAddress' objects internally. 
But you can see exceptions when you use invalid hex string or wrong sized hex string.

File: IconSDK.Tests/TestRPC.cs (hunk 2 above)
Comment (zsaladin) [reply IconSDK.Tests/TestRPC.cs:32]: Yes you are right. I know exceptions are not specified. All exceptions are default exception(System.Exception) for now. I will modify them to custom or proper exceptions someday. I have priorities for implementing features.
But your opinion is very reasonable. This SDK is unofficial and in developing.

**Comment by zsaladin** [comment]
Thank you for your interest. I replied your comments. If you have more questions please feel free to ask.

**Pull request closed without merging**